src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala

```diff
@@ -264,7 +264,9 @@ final case class ClientsApiServiceImpl(
         limit = limit
       )
       hasKeys           <- Future
-        .traverse(pagedResults.results.map(_.id))(authorizationProcessService.getClientKeys)
+        .traverse(pagedResults.results.map(_.id))(id =>
+          authorizationProcessService.getClientKeys(id, relationshipsUuid)
```

Comment:
checking with the frontend, the correct behaviour is 
```suggestion
          authorizationProcessService.getClientKeys(id, Seq.empty)
```